doc/old/getting_started.rst

```diff
@@ -49,7 +49,7 @@ Pydicom is easy to install and use, and because it is a pure Python package, it
 should run wherever Python runs.
 
 One limitation of pydicom: compressed pixel data (e.g. JPEG) can only be
```

Comment:
I mostly left the current docs alone to focus on the API (with the exception of `base_element.rst` because I wanted an example of what my changes would look like). I will get to them though.